group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -8539,5 +8379,20 @@ private static class RebalanceResult {
             this.followerAssignment = followerAssignment;
         }
     }
+
+    private static class SyncResult {
```

Comment:
nit: Should we put this one close to the `RebalanceResult` class. They are quite similar.

Reply:
It's right below RebalanceResult class. did you want to move it elsewhere?

Reply:
Nope. I misread it. Sorry.